
---
bool answer_received
string status


